components/icon-picker/icon-picker.js

```diff
@@ -121,15 +113,16 @@ IconPicker.propTypes = {
 const IconLabel = (props) => {
 	const { icon, isChecked } = props;
 	return (
-		<>
-			<StyledIconButton
-				selected={isChecked}
-				key={icon.name}
-				name={icon.name}
-				iconSet={icon.iconSet}
-			/>
-			<VisuallyHidden>{icon.label}</VisuallyHidden>
-		</>
+		<Tooltip text={icon.label}>
+			<div>
+				<StyledIconButton
+					selected={isChecked}
+					key={icon.name}
+					name={icon.name}
+					iconSet={icon.iconSet}
+				/>
+			</div>
```

Comment:
The reason I add a `div` here is that the `ToolTip` component tries to clone the element passed in in order to add a new ref to it. This fails on the styled components element because it doesn't forward the ref. 

So essentially if there is a better solution for passing one single element here that forwards the ref I'm all for it :)